test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -401,62 +438,96 @@ export default class PerfCompare extends Component {
+                    // raw values displaying: master jobs diplay all their children raw values, and child jobs diplay their own raw values.
+                    curRawValues = {
+                        "baseline": {
+                            "Overview": parsedVariantsBaseline.metrics[j].value,
+                            "Raw Data": (parsedVariantsBaseline.testsData === undefined) ? childBuildBaselineData : this.state.benchmarkRuns.benchmarkRunBaseline.parsedVariants[i].testsData[j].value
+                         },
+                        "test": {
+                            "Overview": parsedVariantsTest.metrics[curMatchingTestMetricIndex].value,
+                            "Raw Data": (parsedVariantsTest.testsData === undefined) ? childBuildTestData : parsedVariantsTest.testsData[j].value
+                         }
+                    };
+                } else {
+                    // perffarm builds
+                    let runBaseline = this.state.benchmarkRuns.benchmarkRunBaseline;
+                    let runTest = this.state.benchmarkRuns.benchmarkRunTest
+                    curRawValues = {
+                        "baseline": runBaseline.parsedVariants[i].metrics[j].value,
+                        "test": runTest.parsedVariants[curMatchingTestVariantIndex].metrics[curMatchingTestMetricIndex].value
+                    };
+                }
 
                 try {
-                    curBaselineScore = math.round((this.state.benchmarkRuns.benchmarkRunBaseline.parsedVariants[i].metrics[j].mean), 3);
+                    curBaselineScore = math.round((parsedVariantsBaseline.metrics[j].value.mean), 3);
                 } catch (roundBaselineScoreError) {
                     curBaselineScore = "error";
                 }
                 
                 try {
-                    curTestScore = math.round((this.state.benchmarkRuns.benchmarkRunTest.parsedVariants[curMatchingTestVariantIndex].metrics[curMatchingTestMetricIndex].mean), 3);
+                    curTestScore = math.round((parsedVariantsTest.metrics[curMatchingTestMetricIndex].value.mean), 3);
                 } catch (roundTestScoreError) {
                     curTestScore = "error";
                 }
 
                 try {
-                    curBaselineCI = math.round((this.state.benchmarkRuns.benchmarkRunBaseline.parsedVariants[i].metrics[j].ci), 3);
+                    curBaselineCI = math.round((parsedVariantsBaseline.metrics[j].value.CI), 3);
                     curBaselineCI += "%";
                 } catch(e) {
                     curBaselineCI = "Not found";
                 }
 
                 try {
-                    curTestCI = math.round((this.state.benchmarkRuns.benchmarkRunTest.parsedVariants[curMatchingTestVariantIndex].metrics[curMatchingTestMetricIndex].ci), 3);
+                    curTestCI = math.round((parsedVariantsTest.metrics[curMatchingTestMetricIndex].value.CI), 3);
```

Comment:
Please do the number format in column render. There is no need to do it for each number. We can use a function. You can refer the following as an example: 
https://github.com/AdoptOpenJDK/openjdk-test-tools/blob/7bd24e101fa667ea601e8094be4c50277e60bc18/test-result-summary-client/src/Build/TopLevelBuilds.jsx#L162